codespell_lib/data/dictionary.txt

```diff
@@ -45200,29 +45207,90 @@ vaguaries->vagaries
 vai->via, vie,
 vaiable->variable
 vaiables->variables
+vaiance->variance
```

Comment:
A suggestion for `vaiance` and friends:
```
vaiance->variance, valiance,
```
Also `valance` but it's probably not common enough, is it?

Reply:
Added `vaiant->valiant` and similar.